### ISSUE TITLE
Link to YouTube video

### ISSUE THREAD

**Issue by rabernat** [xgcm/xgcm#269]
@paigem mentioned that we should link to this xgcm video from SciPy in the docs

https://www.youtube.com/watch?v=lo6JvrJ-YA8&ab_channel=Enthought

**Comment by jbusecke** [comment]
Yes, that is a great idea. Ill group that with the `quickstart guide`(#268)?
